media/css/firefox/nightly-firstrun.scss

```
@@ -20,7 +20,7 @@ main {
     padding-bottom: 0;
 
     .mzp-c-hero-title {
-        @include at2x('#{$image-path}/logos/firefox/browser/nightly/logo-lg.png', 124px, 124px);
+        @include at2x('#{$image-path}/logos/firefox/browser/nightly/logo-lg-high-res.png', 124px, 124px);
```

This mixin automatically generates CSS with the given file as a default used for all screens and a high-res file for higher resolution screens as a secondary rule in a media query. The mixin will append the "-high-res" suffix. So what you've done here is instruct it to use a file named "logo-lg-high-res-high-res.png" for high resolution displays, and that file doesn't exist so this won't work.

To fix this bug we should instead NOT use the at2x mixin and explicitly use the high-res image for all screens.

```suggestion
        background-image: url('/logos/firefox/browser/nightly/logo-lg-high-res.png');
        background-size: 124px 124px;
```